package.json

```diff
@@ -33,7 +33,9 @@
 		"test": "echo \"Error: no test specified\" && exit 1"
 	},
 	"dependencies": {
-		"html-webpack-plugin": "^5.5.4",
+		"archimate-js": "^0.0.4",
```

Comment:
this dependency seems to be unused

Reply:
Is for an other feature, can be removed here.